client/galaxy/scripts/mvc/citation/citation-model.js

```diff
@@ -14,9 +14,9 @@ var logNamespace = 'citation';
 var Citation = Backbone.Model.extend( baseMVC.LoggableMixin ).extend( {
     _logNamespace : logNamespace,
 
-    initialize: function( ) {
-        var bibtex = this.attributes.content;
-        var entry = new bibtex.BibtexParser(bibtex).entries[0];
+    initialize: function() {
+        var bibtex = this.get( 'content' );
+        var entry = BibtexParser(bibtex).entries[0];
```

Comment:
I am getting `Uncaught TypeError: BibtexParser is not a function` here when trying to save a WF with citations.